rootfs/scheduler/resources/__init__.py

```diff
@@ -0,0 +1,12 @@
+from scheduler.resources.__resource import Resource  # noqa
+
+# Load in all resources
+import pkgutil
+import importlib
+import os
+pkgpath = os.path.dirname(__file__)
+for _, name, _ in pkgutil.iter_modules([pkgpath]):
```

Comment:
I think this would read a little better without the `continue` and blank line:
```python
for _, name, _ in pkgutil.iter_modules([pkgpath]):
    if not name.startwswith('__'):
        importlib.import_module('.{}'.format(name), 'scheduler.resources')
```